.github/scripts/tkn_push_bundles.sh

```diff
@@ -14,18 +14,24 @@ printf 'bundle_dirs: %s\n' "${!bundle_dirs[@]}"
 
 # Main loop
 printf 'Pushing any new/modified bundle definitions.\n'
-for BUNDLE_DIR in "${!bundle_dirs[@]}"
+for VERSION_DIR in "${!bundle_dirs[@]}"
 do
-  if [[ ! -d "$BUNDLE_DIR" ]]
+  if [[ ! -d "$VERSION_DIR" ]]
   then
-    printf 'Skipping non-existing bundle-path: %s\n' "$BUNDLE_DIR"
+    printf 'Skipping non-existing bundle-path: %s\n' "$VERSION_DIR"
     continue
   fi
   BUNDLE_FILES=()
+  BUNDLE_DIR=${VERSION_DIR%/*}
   BUNDLE_NAME=${BUNDLE_DIR##*/}
-  printf '* Bundle_dir: %s\n' $BUNDLE_DIR
+  RESOURCE_DIR=${BUNDLE_DIR%/*}
+  RESOURCE_TYPE=${RESOURCE_DIR#*/}
+  printf '* Version dir: %s\n' $VERSION_DIR
+  printf '* Bundle dir: %s\n' $BUNDLE_DIR
   printf '* Bundle name: %s\n' $BUNDLE_NAME
-  for file in $BUNDLE_DIR/*.yaml
+  printf '* Resource dir: %s\n' $RESOURCE_DIR
+  printf '* Resource type: %s\n' $RESOURCE_TYPE
+  for file in $VERSION_DIR/*.yaml
```

Comment:
This could probably change.
In Tekton Catalog the YAML file **MUST** be the exact same as the parent folder.
For example:
`catalog/task/foo  --> catalog/task/foo/0.1/foo.yaml`

So no need to glob anymore. The idea of multiple YAMLs would end here.

Thoughts?

Reply:
I went ahead and made this change. 
There cannot be multiple YAML files anymore, per Tekton Catalog spec.